src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -345,15 +367,21 @@ export class KernelsDropdown extends SelectBox {
 
 export class AttachToDropdown extends SelectBox {
 	private model: NotebookModel;
+	private kernelsDropdown: KernelsDropdown;
```

Comment:
let's chat more here. The AttachToDropdown shouldn't have the KernelsDropdown as a property here.

The KernelsDropdown should react to an event fired from the NotebookModel insteaad of us calling a method on KernelsDropdown here.

Reply:
Once I made the changes to the notebookModel I was able to then utilize the changeKernel method there instead.